test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1352,8 +1352,8 @@ UCS_TEST_P(test_ucp_sockaddr_different_tl_rsc, unset_devices_and_communicate)
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_sockaddr_different_tl_rsc, all, "all")
 
 class test_ucp_sockaddr_cm_switch : public test_ucp_sockaddr {
-    protected:
-    ucp_rsc_index_t get_num_cms()
+protected:
+    const ucp_rsc_index_t get_num_cms()
```

Comment:
`ucp_rsc_index_t get_num_cms() const`

Reply:
/__w/1/s/contrib/../test/gtest/ucp/test_ucp_sockaddr.cc: In member function ‘ucp_rsc_index_t test_ucp_sockaddr_cm_switch::get_num_cms() const’:
/__w/1/s/contrib/../test/gtest/ucp/test_ucp_sockaddr.cc:1358:47: error: passing ‘const test_ucp_sockaddr_cm_switch’ as ‘this’ argument discards qualifiers [-fpermissive]
         const ucp_worker_h worker    = sender().worker();


Removing `const`